Gems/AtomLyIntegration/EMotionFXAtom/Code/Tools/EMStudio/AtomRenderPlugin.h

```diff
@@ -83,6 +88,10 @@ namespace EMStudio
         AZStd::shared_ptr<AzToolsFramework::ManipulatorManager> m_manipulatorManager;
         AZ::Transform m_mouseDownStartTransform;
 
+        // Atom performance metrics
+        QTimer m_metricsTimer;
+        AZStd::string m_fpsStr;
```

Comment:
uber nit: variable name could not use the abbreviation and be `m_fpsDisplay` or `m_fpsOutput` or something instead?